blog-service/2024-04-26-apps.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Classic Apps to Next-Gen Apps Migration (Apps)
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - apps
+  - app catalog
+  - classic-apps
+  - nex-gen-apps
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're happy to announce the migration of 134 [Classic Apps (Legacy)](/docs/get-started/apps-integrations/#classic-apps-legacy) to [Next-Gen Apps](/docs/get-started/apps-integrations/#next-gen-apps). Next-Gen apps simplify the management and maintenance of apps by supporting app versioning and upgrades. These Next-Gen apps can be installed via the App Catalog UI and [Sumo Logic terraform provider](https://registry.terraform.io/providers/SumoLogic/sumologic/latest/docs/resources/app). To view all the available Next-Gen apps select the **Next-Gen Apps** filter in the App Catalog page.
```

Comment:
I would clarify that this is a partially complete migration (more apps to come). I put down 300 total as a placeholder, so you'll need to change it. @JV0812 @himanshu219 

```suggestion
We are happy to announce the partial migration of 134 [Classic Apps (Legacy)](https://chatgpt.com/docs/get-started/apps-integrations/#classic-apps-legacy) to [Next-Gen Apps](https://chatgpt.com/docs/get-started/apps-integrations/#next-gen-apps), out of an expected total of approximately 300.

Next-Gen Apps streamline app management and maintenance by supporting versioning and upgrades. These apps can be installed through the App Catalog UI and the [Sumo Logic Terraform provider](https://registry.terraform.io/providers/SumoLogic/sumologic/latest/docs/resources/app). To explore all available Next-Gen Apps, select the Next-Gen Apps filter on the **App Catalog** page.
```